src/js/dynload.ts

```diff
@@ -129,42 +35,31 @@ export class DynlibLoader {
    *
    * @param lib The file system path to the library.
    * @param global Whether to make the symbols available globally.
-   * @param searchDirs Directories to search for the library.
    * @private
    */
-  public async loadDynlib(lib: string, global: boolean, searchDirs?: string[]) {
+  public async loadDynlib(lib: string, global: boolean) {
     const releaseDynlibLock = await this._lock();
 
     DEBUG &&
       console.debug(`Loading a dynamic library ${lib} (global: ${global})`);
 
-    const fs = this.createDynlibFS(lib, searchDirs);
-    const localScope = global ? null : {};
+    let flags = RTLD_NOW;
+    if (global) {
+      flags |= RTLD_GLOBAL;
+    } else {
+      flags |= RTLD_LOCAL;
+    }
```

Comment:
This does nothing?
```suggestion
    }
```

Reply:
Yes, but I think it is better to be explicit.

Reply:
What about a comment? I think it's weird to have this arithmetic expression that's a no-op. Does the C header also define `RTLD_LOCAL` as 0?

Reply:
> Does the C header also define RTLD_LOCAL as 0?

Yes it does. I copied the values from [dlfcn.h](https://github.com/emscripten-core/emscripten/blob/d45b3957321d6a43cd96f380ba603381ccc4db06/system/lib/libc/musl/include/dlfcn.h#L10-L15).

Anyway, I also noticed that we are not using the `global` flag anymore, so there isn't much reason to define those constants. I changed to use the number directly and added a comment.